client/src/actions/fetchProduct.js

```diff
@@ -0,0 +1,21 @@
+import changeProduct from './productOverview/currentProduct';
+import changeStyles from './productOverview/styleList';
+
+let fetchData = () => {
+
+  return (dispatch) => {
+    fetch('http://127.0.0.1:3000/products', {
+      method: 'GET'
```

Comment:
not important just curious, I think `fetch` by default will send a GET request, are we stating the method here for readability?

Reply:
I didn't know that haha, thanks for the heads up. Happy to keep it there for readability but since I'm not using any other options I think I'll get rid of it.